### ISSUE TITLE
Training Set of WebArena-Lite

### ISSUE THREAD

**Issue by weizhepei** [THUDM/VisualAgentBench#18]
@Xiao9905 Hi Xiao, congratulations for the great work! As stated in the paper (Section D.1), there are 1,186 manually annotated training samples along with reward functions for WebArena-Lite:
> 5 authors create around 40 task prototypes with corresponding solvers, and generating 1,186 valid training samples (i.e., instruction, trajectory, and reward function) for WebArena-Lite.

Now that the paper is accepted to ICLR 2025, is there any plan to release the data? Thank you!
